### PR TITLE
Update Content

### PR DESCRIPTION
**Pull request by leerogers-bjss**
Sat down with luke and done a full content sweep

lots of updates to spelling and rewriting of some bad content.

will need a second pass

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/ListPrice/EditListPrice.cshtml [hunk 1]
```diff
@@ -42,7 +42,7 @@
                         <nhs-select asp-for="DeclarativeTimeUnit"
                                     asp-items="timeUnitsSelectList"
                                     label-text="Unit of time"
-                                    label-hint="Provide the unit of time for how this Catalogue Solution will be billed." />
+                                    label-hint="Select the billing period for this list price." />
```
Comment (bjss-colin) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/ListPrice/EditListPrice.cshtml:45]: Does this need to he ported over to the Additional and Associated Services list price pages?

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/PublicationStatus.cs [hunk 2]
```diff
@@ -5,19 +5,19 @@ namespace NHSD.GPIT.BuyingCatalogue.EntityFramework.Catalogue.Models
 {
     public enum PublicationStatus
```
Comment (bjss-colin) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/PublicationStatus.cs:6]: Looks like these will change wording of the Publication Status list on Solution, Associated Service and Additional Service management pages

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/PublicationStatus.cs (hunk 2 above)
Comment (leerogers-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/PublicationStatus.cs:6]: this is intentional

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/ListPrice/EditListPrice.cshtml (hunk 1 above)
Comment (bjss-colin) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/ListPrice/EditListPrice.cshtml:45]: Views are duplicated at the moment for additional and associated services. We essentially have 3x3 cshtml files for list prices.

Haven't yet gotten around to making them generic.

Comment (leerogers-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/ListPrice/EditListPrice.cshtml:45]: done that

**Pull request merged**